internal/pkg/fleetapi/enroll_cmd.go

```diff
@@ -24,19 +24,19 @@ import (
 type EnrollType string
 
 // ErrTooManyRequests is received when the remote server is overloaded.
-var ErrTooManyRequests = errors.New("too many requests received (429)")
+var ErrTooManyRequests = errors.New("Received status 429(TooManyRequest)")
```

Comment:
I kinda like this guideline 🙂 https://go.dev/wiki/CodeReviewComments#error-strings 

> Error Strings
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.